### PR TITLE
Switch to downstream fork of cloudkitty

### PR DESCRIPTION
**Pull request by jovial**
This brings in a few additional features not in the regular wallaby release.

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -181,6 +181,13 @@ kolla_build_blocks:
                -e '/#baseurl.*/a baseurl={{ repo.url }}' /etc/yum.repos.d/{{ repo.file }}{% if not loop.last %} &&{% endif %} \
     {% endfor %}
 
+  cloudkitty_base_footer: |
+    # Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
+    # Support customising Prometheus queries: https://review.opendev.org/c/openstack/cloudkitty/+/817275
+    RUN pip install git+https://github.com/stackhpc/cloudkitty.git@stackhpc/wallaby
+  horizon_footer: |
+    # Fixes CloudKitty pricing panel: https://github.com/stackhpc/cloudkitty-dashboard/pull/1
+    RUN pip install git+https://github.com/stackhpc/cloudkitty-dashboard.git@stackhpc/wallaby
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: Could you use the kolla_sources approach rather than footers?

Comment (jovial) [reply etc/kayobe/kolla.yml]: Oh yeah, missed that thanks.

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -104,6 +104,17 @@ kolla_sources:
     type: git
     location: "{{ kolla_bifrost_source_url }}"
     reference: "{{ kolla_bifrost_source_version }}"
+  cloudkitty-base:
+    # Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
```
Comment (jovial) [inline etc/kayobe/kolla.yml]: Not sure if I should bother maintaining these comments here as they may get out of sync with the branch. Thoughts?

Comment (markgoddard) [reply etc/kayobe/kolla.yml]: They could go stale, maybe leave them out.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**